src/core/annotation.js

```diff
@@ -602,6 +602,14 @@ var WidgetAnnotation = (function WidgetAnnotationClosure() {
       this.setFlags(AnnotationFlag.HIDDEN);
     }
 
+    // Support ResetForm Action Widget
+    if (data.fieldType === 'Btn' && dict.get('A') !== undefined) {
+      var actionRef = dict.map.A.num;
+      var xrefObject = dict.xref.fetchIfRef(dict.map.A);
+      data.formAction = {};
```

Comment:
No, this is not an unused variable. See line 610. I am then using this code to determine what type of annotation this is. Example code:

`if (item.formAction.S === "ResetForm") {`